src/jace/stages.py

```diff
@@ -166,14 +165,21 @@ def lower(self, *args: _P.args, **kwargs: _P.kwargs) -> JaCeLowered[_R]:
         trans_ctx: translator.TranslationContext = builder.translate_jaxpr(jaxpr)
 
         flat_call_args = jax_tree.tree_leaves((args, kwargs))
+        device = util.parse_backend_jit_option(self._jit_options.get("backend", "cpu"))
```

Comment:
Shouldn't the default backend be defined in a single place and then used from where is needed instead of as a hardcoded string in multiple places?

Reply:
You are right.
I have now introduced a global variable, which contains the string.
I was also able to reduce it to a single point where it is used/needed.